CONTRIBUTING.md

```diff
@@ -95,6 +95,22 @@ Try to keep the changes focused. If you submit a large amount of work in all in
 
 #### 4. Submit a [pull request](https://help.github.com/articles/about-pull-requests/)
 
+Please keep the title of your pull request short but informative - it will 
+appear in the changelog.
```

Comment:
Let's link to the changelog to encourage contributors to look at model changelog entries.

Reply:
thx! done